runtime/lua/vim/treesitter/playground.lua

```diff
@@ -269,6 +269,7 @@ function M.inspect_tree(opts)
   vim.bo[b].buflisted = false
   vim.bo[b].buftype = 'nofile'
   vim.bo[b].bufhidden = 'wipe'
+  vim.b[b].query_lint_on = {} -- disable query linter
```

Comment:
I wouldn't mirror the global variable here; just use a bool that allows skipping completely -- I see no point in buffer-local events?

Reply:
renamed to `b:disable_query_linter`